### ISSUE TITLE
NullPointerException in DisksImpl::wrapModel

### ISSUE THREAD

**Issue by m-vojvodic** [Azure/azure-sdk-for-java#1550]
Attempting to find a deleted disk results in a null pointer exception:
```
java.lang.NullPointerException
     at com.microsoft.azure.management.compute.implementation.DisksImpl.wrapModel(DisksImpl.java:83)
     at com.microsoft.azure.management.compute.implementation.DisksImpl.getByGroup(DisksImpl.java:60)
     at com.microsoft.azure.management.compute.implementation.DisksImpl.getByGroup(DisksImpl.java:20)
```
This behavior differs from the implementations of `wrapModel` for `NetworksImpl`, etc., which have a null pointer check.

**Comment by anuchandy** [comment]
Thanks @m-vojvodic, yes it a bug - assigning to me

**Comment by martinsawicki** [comment]
fixed in master, which will be released as v1.0.0 soon.
